data/import_data.py

```diff
@@ -94,12 +94,19 @@ def import_state(path):
             scores = []
             for criterion in Criterion.query.filter_by(category_id=category.id).all():
                 meets_criterion = criterion.id in category_data['criteria_met']
+                maybe_meets_criterion = criterion.id in category_data['criteria_maybe_met']
```

Comment:
It took me longer than I'd like to admit to figure out that `category_data['criteria_met']` and `category_data['criteria_maybe_met']` were lists of IDs. Maybe it's because I just woke up or because I barely have backend experience, but it might be worth it to make these lines even more beginner-friendly
```suggestion
                criteria_met_ids = category_data['criteria_met']
                criteria_maybe_met_ids = category_data['criteria_maybe_met']
                meets_criterion = criterion.id in criteria_met_ids
                maybe_meets_criterion = criterion.id in criteria_maybe_met_ids
```
But then again, this looks inelegant, so I don't know, I could go for either option. Up to you

Reply:
Good suggestion! Thank you!